Libraries/Docs/DocsCommentsContainer.cs

```diff
@@ -237,15 +237,15 @@ private void LoadFile(FileInfo fileInfo)
             if (add)
             {
                 int totalMembersAdded = 0;
-                Types.Add(docsType);
+                Types.TryAdd(docsType.DocId, docsType); // is it OK this encounters duplicates?
```

Comment:
A lurking bug for sure, but one that would likely cause issues for us elsewhere too. The docs site would probably struggle if duplicates existed. We could benefit from a "validate" mode in this tool that would report on any such issues; we could then define the behavior for invalid content.

Reply:
#79 filed